test/integration/api/services_controller_test.rb

```diff
@@ -14,6 +14,10 @@ class Api::ServicesControllerTest < ActionDispatch::IntegrationTest
 
   class SettingsTest < Api::ServicesControllerTest
     test 'settings renders the right template and contains the right sections' do
+      Account.any_instance.stubs(:provider_can_use?).returns(true)
+      Account.any_instance.stubs(:provider_can_use?).with(:api_as_product).returns(false | true)
```

Comment:
Do you need this line? What for? I tried without it and it seems to work perfectly well:
![image](https://user-images.githubusercontent.com/11318903/65620411-16c26980-dfc2-11e9-8ae1-f0c30bc46423.png)